applications/FemToDemApplication/CMakeLists.txt

```diff
@@ -5,54 +5,60 @@ message("**** configuring KratosFemToDemApplication ****")
 ################### PYBIND11
 include(pybind11Tools)
 
-include_directories( ${CMAKE_SOURCE_DIR}/kratos )
-include_directories( ${CMAKE_SOURCE_DIR}/applications/SolidMechanicsApplication )
-include_directories( ${CMAKE_SOURCE_DIR}/applications/DEMApplication )
-include_directories( ${CMAKE_SOURCE_DIR}/applications/MeshingApplication )
+include_directories(${CMAKE_SOURCE_DIR}/kratos )
+include_directories(${CMAKE_SOURCE_DIR}/applications/SolidMechanicsApplication)
+include_directories(${CMAKE_SOURCE_DIR}/applications/DEMApplication)
+include_directories(${CMAKE_SOURCE_DIR}/applications/MeshingApplication)
+include_directories(${CMAKE_SOURCE_DIR}/applications/PfemFluidDynamicsApplication)
+include_directories(${CMAKE_SOURCE_DIR}/applications/DelaunayMeshingApplication)
 
 # generate variables with the sources
 set( KRATOS_FEM_TO_DEM_APPLICATION_SOURCES
-	${CMAKE_CURRENT_SOURCE_DIR}/fem_to_dem_application.cpp
-  	${CMAKE_CURRENT_SOURCE_DIR}/fem_to_dem_application_variables.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/fem_to_dem_python_application.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_utilities_to_python.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_constitutive_laws_to_python.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_strategies_to_python.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_processes_to_python.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/zarate_law.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/fem_dem_elastic_law.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/generic_small_strain_femdem_element.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/generic_large_displacement_femdem_element.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/extend_pressure_condition_process.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/update_pressure_value_pfem_conditions_process.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/assign_pressure_id_process.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/expand_wet_nodes_process.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/regenerate_pfem_pressure_conditions_process.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/generate_dem_process.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/update_dem_kinematics_process.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/transfer_nodal_forces_to_fem.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/compute_normalized_free_energy_on_nodes_process.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/compute_sand_production.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/fix_free_velocity_on_nodes_process.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/remove_alone_DEM_elements_process.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/constitutive_law_utilities.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/fem_to_dem_application.cpp

```

Comment:
I recommend to follow the structure of the StructuralMechanicsApplication. It is way simpler